src/config.rs

```diff
@@ -33,6 +33,41 @@ impl Display for OverrideReason {
     }
 }
 
+#[derive(Debug)]
+pub enum PgpKey {
```

Comment:
should probably document if the intention is for these to be public or secret keys

Reply:
I agree, I'll push a change momentarily to rename that to `PgpPublicKey` to make it clearer.